projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -46,28 +47,47 @@ class Notebook:
     Other cells are ignored.
     """
 
-    def __init__(self, file_path: Path):
+    @staticmethod
+    def register_notebook_steps(file_path: Path, context: JobContext):
         try:
-            self.sql_and_run_cells = []
-            self.python_helper_cells = []
-            self.file_path = file_path
+            python_module = imp.new_module("notebook")
+            exec("job_input = 1", python_module.__dict__)
```

Comment:
Can you explain? 
what does job_input = 1 mean?

Reply:
see this https://github.com/vmware/versatile-data-kit/pull/1520#discussion_r1081285633 
I will put a comment there